Jenkinsfile

```diff
@@ -95,7 +95,7 @@ def androidUnitTests(nodeVersion, npmVersion, testOnDevices) {
 								sh label: 'Run Test Suite on device(s)', script: "npm run test:integration -- android -T device -C all"
 							// run PR tests on emulator
 							} else {
-								sh label: 'Run Test Suite on emulator', script: "npm run test:integration -- android -T emulator -D test -C android-28-playstore-x86"
+								sh label: 'Run Test Suite on emulator', script: "npm run test:integration -- android -T emulator -D test -C android-30-playstore-x86"
```

Comment:
Can we use an x86_64 emulator image instead?

Reply:
We only install `x86` on the nodes atm

Reply:
I can certainly modify our chef scripts to set up x86_64 emulators. I'll see if I can't do that now...